src/blocks/remote-data-container/components/modals/InputModal.tsx

```diff
@@ -60,7 +60,7 @@ export function InputModal( props: InputModalProps ) {
 						style={ { marginBottom: '8px' } }
 					/>
 				) ) }
-				<Button variant="primary" onClick={ onSelectItem }>
+				<Button variant="primary" onClick={ onSelectItem } type="submit">
```

Comment:
We should probably remove `onClick` here and do it in the form `onSubmit` with `e.preventDefault();` there, unless I'm missing something. Although this probably does the job well enough.

Reply:
I don't believe it makes too much of a difference since the button is inside a form. But I can see how it makes more sense having an `onSubmit` instead. Updated here: [`2146c94` (#301)](https://github.com/Automattic/remote-data-blocks/pull/301/commits/2146c942a5168dbcf00353ca7a566f301f93ecfd)

Reply:
Yeah, it shouldn't make much of a difference. In the event that the form was ever submitted another way though, this would work properly now. I don't know if there is a way that can happen with screen readers or other accessibility tools or anything... but probably better to use the proper form submission mechanism either way..